packages/babel-preset-default/index.js

```diff
@@ -3,31 +3,31 @@ module.exports = function( api ) {
 
 	return {
 		presets: [
-			! isTestEnv && [ '@babel/preset-env', {
+			! isTestEnv && [ require.resolve( '@babel/preset-env' ), {
```

Comment:
I did some research and I found that `babel-loader` has some issues with the way how paths are being resolved and this is one of the workarounds often shared by the community. However, if we go this route we might need to update more places in our codebase. One of them is here:
https://github.com/WordPress/gutenberg/blob/master/bin/packages/get-babel-config.js#L16

Ideally, we don't have to change anything in the preset itself.

Reply:
I've found a way to make this work for 3rd parties and Gutenberg. TLDR is that this proposal centralizes all babel logic into the `@wordpress/babel-preset-default`.